src/main/java/nextstep/qna/domain/Question.java

```diff
@@ -35,54 +34,75 @@ public Question(Long id, NsUser writer, String title, String contents) {
         this.writer = writer;
         this.title = title;
         this.contents = contents;
+        this.answers = Answers.init();
     }
 
     public Long getId() {
         return id;
     }
 
-    public String getTitle() {
-        return title;
+    public NsUser getWriter() {
+        return writer;
     }
 
-    public Question setTitle(String title) {
-        this.title = title;
-        return this;
+    public void addAnswer(Answer answer) {
+        answer.toQuestion(this);
+        this.answers = answers.add(answer);
     }
 
-    public String getContents() {
-        return contents;
+    public List<DeleteHistory> deleted(NsUser loginUser) {
+        validateDelete(loginUser);
+        delete();
+        return deletedHistory();
     }
 
-    public Question setContents(String contents) {
-        this.contents = contents;
-        return this;
+    public void validateDelete(NsUser loginUser) {
```

Comment:
```suggestion
    private void validateDelete(NsUser loginUser) {
```
DeleteHistory 를 생성하기 위한 검증 로직이네요 public 으로 노출 시킬 필요는 없을 것 같습니다.